test/unit/gemcutter/middleware/hostess_test.rb

```diff
@@ -5,6 +5,7 @@ class Gemcutter::Middleware::HostessTest < ActiveSupport::TestCase
   include Rack::Test::Methods
 
   setup do
+    RubygemFs.mock!
```

Comment:
Any info why this is needed?

Reply:
I had a trouble with it trying to use the S3 mock instead of Local. It was transient, based on another change, but I didn't like that this directly tests an object that uses RubygemFs but didn't explicitly make sure it was mocking before each test.